Editor/src/Windows/WelcomeWindow.cpp

```diff
@@ -33,6 +34,16 @@ namespace Nuake
 			nlohmann::json projectJson = nlohmann::json::parse(projectFile);
 			Name = projectJson["Name"];
 			Description = projectJson["Description"];
+
+			const std::string projectIconPath = Path + "/../icon.png";
+			if (FileSystem::FileExists(projectIconPath, true))
+			{
+				ProjectIcon = TextureManager::Get()->GetTexture(projectIconPath);
+			}
+			else
+			{
+				ProjectIcon = TextureManager::Get()->GetTexture("resources/Images/nuake-logo.png");
+			}
```

Comment:
I don't know if your prefer it this way, but I'm just suggesting how I'm usually doing it when there's a default value and when not using Ternary Operator(which in this case would make the line too long and hard to read)

```suggestion
            ProjectIcon = TextureManager::Get()->GetTexture("resources/Images/nuake-logo.png");
			if (FileSystem::FileExists(projectIconPath, true))
			{
				ProjectIcon = TextureManager::Get()->GetTexture(projectIconPath);
			}
```